<launch>

    <node name="move_arm_server" pkg="arm_master" type="move_arm_server.py" output="screen"/>
    <node name="main" pkg="arm_master" type="arm_master_main_ITA.py" output="screen"/>
    <node name="brick_manager_server" pkg="brick_manager" type="object_manager_server_ITA.py" output="screen"/>
    <node name="check_dropped" pkg="arm_master" type="dropped_checker_server_ITA.py" output="screen"/>

</launch>
